playbooks/roles/deploy-osh/templates/neutron.yaml.j2

```diff
@@ -1 +1,23 @@
-{{ lookup('template','files/common-images-overrides.yml') | from_yaml | to_nice_yaml(indent=2) }}
\ No newline at end of file
+{% if developer_mode %}
+conf:
+  # api-paste entrypoint neutron.api.versions:Versions.factory was deprecated in Queens
+  # We must replace it for Rocky deployments
+  paste:
+    app:neutronversions:
+      paste.app_factory: neutron.pecan_wsgi.app:versions_factory
+images:
+  tags:
+    bootstrap: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+    db_init: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+    db_drop: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+    ks_user: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+    ks_service: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+    ks_endpoints: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_openstack_image_version }}"
+    neutron_db_sync: "{{ suse_osh_registry_location }}/openstackhelm/neutron:{{ suse_openstack_image_version }}"
+    neutron_dhcp: "{{ suse_osh_registry_location }}/openstackhelm/neutron:{{ suse_openstack_image_version }}"
+    neutron_l3: "{{ suse_osh_registry_location }}/openstackhelm/neutron:{{ suse_openstack_image_version }}"
+    neutron_linuxbridge_agent: "{{ suse_osh_registry_location }}/openstackhelm/neutron:{{ suse_openstack_image_version }}"
```

Comment:
Do you think we should add:
```

    neutron_sriov_agent: docker.io/openstackhelm/neutron:ocata-sriov-1804
    neutron_sriov_agent_init: docker.io/openstackhelm/neutron:ocata-sriov-1804
```

to remember we still need to do those for the technical preview?

Reply:
Well, if it is not replaced it is still the same value.

If we have to remember those, I believe they should be tracked in Jira, not in the code.

Reply:
They should be both tracked in Jira and code. Jira is internal, we don't know who wants to help in here.